test/integration/device-api/actions.spec.ts

```diff
@@ -251,6 +265,8 @@ describe('manages application lifecycle', () => {
 		});
 
 		beforeEach(async () => {
+			// Delete all update locks
+			await updateLock.releaseLock(APP_ID);
```

Comment:
You probably should delete locks afterwards as well, a bug on a test means leftover locks could be left and cause other tests to fail. Those kind of test failures are pretty hard to debug.

Alternatively, you could also use testfs, which also sets up some hooks to run after all tests.

Reply:
Done, used testfs